src/sentry/web/frontend/generic.py

```diff
@@ -33,13 +33,29 @@ def resolve(path):
     return os.path.split(absolute_path)
 
 
+def static_media_with_manifest(request, **kwargs):
```

Comment:
This is largely unnecessary, we could move this into `static_media`, but in this case we lean on django URL routing a bit to parse the URL.